llvm/lib/Target/ARM/ARMLegalizerInfo.h

```diff
@@ -23,7 +23,8 @@ namespace llvm {
 
 class ARMSubtarget;
 
-/// This class provides the information for the target register banks.
+/// This class provides the information for the ARM target legalizer for
```

Comment:
I'd just delete the comment from all the target implementations altogether

Reply:
I delete them.